package.json

```diff
@@ -11,11 +12,11 @@
   "repository": "https://github.com/AtomLinter/linter-markdown",
   "license": "MIT",
   "engines": {
-    "atom": ">=1.0.0 <2.0.0"
+    "atom": "^1.0.0"
```

Comment:
Does this format actually work? Intuitively it _should_, but this is the only documentation I can find on it: https://atom.io/docs/v1.3.0/hacking-atom-package-word-count#publishing